server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/CVRExportUpload.java

```diff
@@ -210,16 +212,17 @@ private void parseAndPersistFile(final UploadInformation the_info) {
     if (the_info.my_ok) {
       try (InputStream cvr_is = new FileInputStream(the_info.my_file)) {
         final InputStreamReader cvr_isr = new InputStreamReader(cvr_is, "UTF-8");
-        final CVRExportParser parser = new DominionCVRExportParser(cvr_isr, county, 
-                                                                   the_info.my_timestamp);
+        final CVRExportParser parser =
+            new DominionCVRExportParser(cvr_isr, county, the_info.my_timestamp);
         if (parser.parse()) {
-          Main.LOGGER.info(parser.parsedIDs().size() + " CVRs parsed from " + county + 
+          Main.LOGGER.info(parser.parsedIDs().size() + " CVRs parsed from " + county +
                            " county upload file");
           final OptionalLong count = count();
           if (count.isPresent()) {
             Main.LOGGER.info(count.getAsLong() + " uploaded CVRs in storage");
           }
-          attemptFilePersistence(the_info.my_file, county, hash, the_info.my_timestamp);
+          attemptFilePersistence(the_response, the_info.my_file, county, hash,
```

Comment:
I think it is fine to be pragmatic and leave it as-is right now, since it is a private method.